modules/Sources/Dependencies/AddressBookClient/AddressBookEncryption.swift

```diff
@@ -146,41 +118,41 @@ extension AddressBookClient {
     ///     [Unencrypted data]        `address book version`
     ///     [Unencrypted data]        `timestamp`
     ///     [Unencrypted data]        `contacts`
-    static func contactsFrom(unencryptedData: Data) throws -> AddressBookContacts {
+    static func contactsFrom(_ data: Data) throws -> AddressBookContacts {
```

Comment:
When I suggested changing this label, it hadn't registered with me that it would increase the confusability with `contactsFrom(encryptedData:)`.

If the label were `plainData:`, that would clearly distinguish them while being accurate for both the case where the data has been decrypted, and the case where it was never encrypted.
```suggestion
    static func contactsFrom(plainData: Data) throws -> AddressBookContacts {
```

Reply:
Why is that distinction relevant here? This is a static function; it doesn't care about the source of the data or its history.

Reply:
Updated